package.json

```diff
@@ -31,7 +31,7 @@
     "inherits": "2.0.3",
     "lodash": "4.17.2",
     "uuid": "3.0.1",
-    "uvm": "1.4.0"
+    "uvm": "1.5.0"
```

Comment:
uvm 1.5.0 has the fixes where any dispatch post disconnection does no longer throw things. instead it triggers an error event.